src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -0,0 +1,816 @@
+                    <div className="p-4 my-8 border border-gray-100 rounded-md bg-gray-50 dark:border-gray-800 dark:bg-gray-800">
+                      <CommunityResource type="course" />
+                    </div>
+                  )}
+                </div>
+              </div>
+            </div>
+            <div className={'flex flex-row grid grid-rows-1'}>
+              <div className="mx-auto ">
+                {/*start of instructor block*/}
+                <section className="my-8">
+                  <h2 className="text-xl font-bold mb-4 sm:mx-0 mx-auto w-fit">
+                    Meet the Instructor
+                  </h2>
+                  <div className="flex flex-row justify-between">
+                    <div>
+                      <div className="flex justify-center md:hidden mb-4">
+                        <Image
+                          className="rounded-full ml-4 justify-center"
+                          src={avatar_url}
+                          layout="fixed"
+                          width="200"
+                          height="200"
+                          alt={`${name}'s avatar`}
+                        />
+                      </div>
+                      <h2 className="font-semibold text-lg pb-4 sm:mx-0 mx-auto w-fit">
+                        Hi, I'm Mark Shust
+                      </h2>
+                      <p className="mb-2 prose text-gray-900 dark:prose-dark md:prose-lg md:dark:prose-lg-dark dark:text-gray-100 dark:prose-a:text-blue-300 dark:hover:prose-a:text-blue-200 prose-a:text-blue-500 hover:prose-a-:text-blue-600 ">
+                        I have over 20 years of web development experience, and
+                        has been working with PHP for about as long. Soon after
+                        starting development, I became extremely interested and
+                        involved in open source programming, diving into various
+                        PHP frameworks over the years including OSCommerce,
+                        Drupal, and Laravel.
+                      </p>
+                      <p className="mb-6 prose text-gray-900 dark:prose-dark md:prose-lg md:dark:prose-lg-dark dark:text-gray-100 dark:prose-a:text-blue-300 dark:hover:prose-a:text-blue-200 prose-a:text-blue-500 hover:prose-a-:text-blue-600 ">
+                        I'm excited to bring you a comprehensive introduction
+                        course on PHP, and I hope you enjoy it!
+                      </p>
+                    </div>
+                    <div className="flex lg:scale-100 md:scale-75 justify-center items-center hidden md:block xl:pr-12">
+                      <Image
+                        className="rounded-full ml-4 justify-center"
+                        src={avatar_url}
+                        layout="fixed"
+                        width="300"
+                        height="300"
+                        alt={`${name}'s avatar`}
+                      />
```

Comment:
@zacjones93 mostly wrote the instructor introduction. I just adjusted the image's responsiveness

Reply:
I pulled the introduction from Marks site bio